src/coreclr/jit/gentree.cpp

```diff
@@ -16921,6 +16921,55 @@ ssize_t GenTreeIndir::Offset()
     }
 }
 
+/** Test whether the two given nodes are the same leaves.
```

Comment:
could you please update the header to the new format while you are here including // static comment?